src/um/processthreadsapi.rs

```diff
@@ -60,31 +68,32 @@ STRUCT!{struct STARTUPINFOW {
     hStdError: HANDLE,
 }}
 pub type LPSTARTUPINFOW = *mut STARTUPINFOW;
+
 extern "system" {
     pub fn QueueUserAPC(
         pfnAPC: PAPCFUNC,
         hThread: HANDLE,
-        dwData: ULONG_PTR,
+        dwData: ULONG_PTR
```

Comment:
This is done automatically by rustfmt when I was experimenting with it. I think I brought this up in gitter, but I can't remember. I don't see any specific comment on it in CONTRIBUTING. You want trailing commas?